src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -258,7 +256,7 @@ def sync_deploy_root_with_stage(self, role: str) -> DiffResult:
 
         # Does a stage already exist within the app pkg, or we need to create one?
         # Using "if not exists" should take care of either case.
-        log.info("Checking if stage exists, or creating a new one if none exists.")
+        cc.step("Checking if stage exists, or creating a new one if none exists.")
```

Comment:
Do we need to use phases anywhere in our code? For example for application package updates vs application instance updates?

Reply:
Some of our complex workflows could use it, something like run could have a phase for the app package, and a phase for the instance, for example. It might be helpful context for a user to understand what our warnings apply to I guess.